app/controllers/ingredients_controller.rb

```diff
@@ -1,5 +1,5 @@
 class IngredientsController < ApplicationController
-  before_action :set_ingredient, only: [:edit, :update, :show]
+  before_action :set_ingredient, only: [:edit, :update, :show, :destroy] #Added destroy here
```

Comment:
you don't need comment "added destroy here"

Reply:
The comment is gone in the new commit